src/components/Agreement/Agreement.js

```diff
@@ -13,7 +15,14 @@ const Agreement = React.memo(function Agreement() {
       <Split
         primary={
           <>
-            <Box>Details</Box>
+            <Box>
+              <AgreementHeader title="Aragon Network DAO Agreement" />
+              <AgreementDetails
+                contractAddress="0x5c6620c49f9aecf74bd483054f2d0ace0d375f96"
+                creationDate="2020/07/20"
+                ipfsUri="ipfs:Qmb5CHbQQQx6YXkPE6HodeXVmtCRgpSgkj9EkW9xs6jDHj"
```

Comment:
Pulled straight from old mock data, probably old and outdated but will be replaced.

Reply:
Good idea! I think we're pretty close to getting the connector in so perhaps we can shape the data properly as we work on that :)